buildlib/dockers/docker-compose-aarch64.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.4"
+# To build images, run:
+# export ARCH=$(uname -m) && docker compose build
+# docker compose -f docker-compose-${ARCH}.yml build
```

Comment:
can it be 
`docker compose -f docker-compose-$(uname -m).yml build`

Reply:
Changed to fixed ARCH values.